Applications/Utils/OGSFileConverter/OGSFileConverter.cpp

```diff
@@ -45,15 +45,15 @@ void OGSFileConverter::convertGML2GLI(const QStringList &input, const QString &o
     GeoLib::GEOObjects geo_objects;
     GeoLib::IO::XmlGmlInterface xml(geo_objects);
 
-    for (const auto& it : input)
+    for (const auto& string : input)
```

Comment:
No, neither is qstring. I need semantic understanding of that thing to name it correctly...
Maybe Karsten will come up with an idea.